From 38e43e650182b8efdac72ce7325592d36aa78087 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Mon, 26 Nov 2012 14:05:58 +0100
Subject: [PATCH 492/628] usb: otg: usbps_otg: Automatically setup VBUS

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit adb18653d77b5b12c419f43a1efc943cdb03b629

This is zedboard requirement to get otg-host to work.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 include/linux/usb/xilinx_usbps_otg.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/linux/usb/xilinx_usbps_otg.h b/include/linux/usb/xilinx_usbps_otg.h
index 9b1aaf6..8c360b7 100644
--- a/include/linux/usb/xilinx_usbps_otg.h
+++ b/include/linux/usb/xilinx_usbps_otg.h
@@ -166,6 +166,7 @@ struct otg_hsm {
 
 struct xusbps_otg {
 	struct usb_phy		otg;
+	struct usb_phy		*ulpi;
 
 	struct otg_hsm		hsm;
 
-- 
1.7.5.4

